src/__tests__/imagings/search/ImagingRequestTable.test.tsx

```diff
@@ -27,42 +18,34 @@ describe('Imaging Request Table', () => {
     requestedOn: new Date().toISOString(),
     requestedBy: 'some user',
   } as Imaging
-  const expectedImagings = [expectedImaging]
 
   const render = (searchRequest: ImagingSearchRequest) => {
```

Comment:
[minor] some test files call this function `setup` and don't have rename `render as rtlRender` => do we want to migrate to that convention maybe?

Reply:
Yes, I have been trying to hunt these down myself too, any semblance of Enzyme (Wrapper Style Setup, etc...) `setup` is definitely the convention we went with, and should try to keep it consistent. Anyone can pick those up I have only done a few